### PR TITLE
D2IQ-65964: agent_service_account.json needs to be generated.

### PR DESCRIPTION
**Pull request by sergiimatusEPAM**
## High-level description

The hostname for dcos-diagnostics should be the IP address of the host.
The `agent_service_account.json` needs to be generated by `python.exe bootstrap.py dcos-diagnostics-agent` on the Windows node.

## Corresponding DC/OS tickets (required)

  - [D2IQ-65964](https://jira.mesosphere.com/browse/D2IQ-65964)

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by jeschkies** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required internal change

**Comment by janisz** [comment]
We need similar change for telegraph

**Comment by jeschkies** [comment]
@sergiimatusEPAM commented:
To summarize on "dcos-diagnostics" PRs - each of following changes caused an issues:
1) added new record
`"iam-config": "C:\\d2iq\\dcos\\var\\run\\etc\\dcos-diagnostics\\agent_service_account.json" in C:\d2iq\dcos\etc\dcos-diagnostics\dcos-diagnostics-config.json`
Result:
`FATA[0000] Could not init diagnostics job properly: could not init diagnostic job: could not initialize external log providers: could not read /opt/mesosphere/etc/endpoints_config.json: open /opt/mesosphere/etc/endpoints_config.json: The system cannot find the path specified.`
2) added record --hostname 172.16.13.227 in the startup command of the dcos-diagnostics service
Result:
`unknown flag: --hostname`
3) The Bootstrap package in OPEN variant doesn't contain bin/bootstrap file. So we need to decide either make the package unified for OPEN and EE, or make another paackages such as Dcos-diagnostics/Telegraf customized at EE variant and make execution of "python.exe bin/bootstrap dcos-diagnostics-service / dcos-telegraf-service" in EE only

File: packages/dcos-diagnostics/extra/dcos-diagnostics.nssm.j2 [hunk 1]
```diff
@@ -3,7 +3,7 @@ DisplayName = dcos-diagnostics
 Description = DC/OS Diagnostics Windows Agent: exposes component health
 Application = {{ pkg_inst_dpath }}\bin\dcos-diagnostics.exe
 AppDirectory = {{ pkg_inst_dpath }}\bin
-AppParameters = --config {{ dcos_cfg_dpath }}\dcos-diagnostics\dcos-diagnostics-config.json --role=agent daemon
+AppParameters = --config {{ dcos_cfg_dpath }}\dcos-diagnostics\dcos-diagnostics-config.json --role=agent daemon --hostname {{ local_priv_ipaddr }}
```
Comment (jeschkies) [inline packages/dcos-diagnostics/extra/dcos-diagnostics.nssm.j2]:
```suggestion
AppParameters = daemon --config {{ dcos_cfg_dpath }}\dcos-diagnostics\dcos-diagnostics-config.json --hostname {{ local_priv_ipaddr }}
```

Comment (jeschkies) [reply packages/dcos-diagnostics/extra/dcos-diagnostics.nssm.j2]: @sergiimatusEPAM could you accept this change 🙂

**Comment by jeschkies** [comment]
@mesosphere-mergebot bump-ee

**Comment by jongiddy** [comment]
These commits are merged in #7117

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Holding

**Pull request closed without merging**